--- dnf/tests/test_cli.py.orig	2013-02-04 15:26:39.727288265 +0100
+++ dnf/tests/test_cli.py	2013-02-04 15:26:57.317345131 +0100
@@ -57,23 +57,3 @@ class Cli(unittest.TestCase):
         self.cli._configure_repos(opts)
         self.assertTrue(self.yumbase._override_sigchecks)
         self.assertTrue(self.yumbase.repos.getRepo("main")._override_sigchecks)
-
-@mock.patch('dnf.yum.Base.doLoggingSetup', new=mock.MagicMock)
-class TestConfigure(unittest.TestCase):
-    def setUp(self):
-        self.yumbase = base.MockYumBase("main")
-        self.cli = dnf.cli.cli.Cli(self.yumbase)
-
-    def test_configure(self):
-        """ Test Cli.configure.
-
-            For now just see that the method runs.
-        """
-        self.cli.configure(['update'])
-        self.assertEqual(self.cli.cmdstring, "dnf update ")
-
-    def test_configure_verbose(self):
-        self.cli.configure(['-v', 'update'])
-        self.assertEqual(self.cli.cmdstring, "dnf -v update ")
-        self.assertEqual(self.yumbase.conf.debuglevel, 6)
-        self.assertEqual(self.yumbase.conf.errorlevel, 6)
