From 784501a9b3b2c30610e67754ec4aaa2a26096022 Mon Sep 17 00:00:00 2001
From: Jan Silhan <jsilhan@redhat.com>
Date: Wed, 10 Jun 2015 15:11:31 +0200
Subject: [PATCH 2/3] group: fixed installing of already installed environment

---
 dnf/base.py | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/dnf/base.py b/dnf/base.py
index e1b32e6..2be68f2 100644
--- a/dnf/base.py
+++ b/dnf/base.py
@@ -1193,8 +1193,9 @@ class Base(object):
         types = self._translate_comps_pkg_types(types)
         trans = dnf.comps.install_or_skip(solver.environment_install,
                                           env, types, exclude or set())
-        if trans:
-            return self._add_comps_trans(trans)
+        if not trans:
+            return 0
+        return self._add_comps_trans(trans)
 
     def environment_remove(self, env):
         solver = self.build_comps_solver()
-- 
2.4.2

